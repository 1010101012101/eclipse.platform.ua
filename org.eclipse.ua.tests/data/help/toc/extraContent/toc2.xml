<?xml version="1.0" encoding="utf-8"?>
<?NLS  TYPE="org.eclipse.help.toc"?>
<toc label="Samples" topic="samples.htm">
   <topic label="Bats FOR product: Happy Bats, WAS; version: 9.0,8.0 ; platform: z/OS;AIX" href="simple_page.html">
   	  <criteria name="prodname" value="Happy Bats" />
   	  <criteria name="prodname" value="WAS" />
   	  <criteria name="version" value="9.0" />
	  <criteria name="version" value="8.0" />
   	  <criteria name="platform" value="z/OS" />
   	  <criteria name="platform" value="AIX" />
      <topic label="Caring for your fruit bat FOR Version: 9.0; platform: z/OS; product: WAS" href="simple_page.html">
	  	<criteria name="version" value="9.0" />
   	  	<criteria name="platform" value="z/OS" />
   	  	<criteria name="prodname" value="WAS" />
	  </topic>
      <topic label="Cleaning your fruit bat FOR Version: 8.0; platform: AIX; product: Happy Bats" href="simple_page.html">
   	  	<criteria name="platform" value="AIX" />
	  	<criteria name="version" value="8.0" />
	  	<criteria name="prodname" value="Happy Bats" />
   	  </topic>
      <topic label="Feeding your bat" href="simple_page.html"/>
      <topic label="Bat sonar" href="simple_page.html"/>
      <topic label="Bat guano" href="simple_page.html"/>
      <topic label="Bat history" href="simple_page.html"/>
      <topic label="Bat recycling" href="simple_page.html"/>
  </topic>
	<topic label="Bats FOR product: Happy Bats" href="simple_page.html">
		  <criteria name="prodname" value="Happy Bats" />
	      <topic label="Feeding your bat" href="simple_page.html"/>
	      <topic label="Bat sonar" href="simple_page.html"/>
	      <topic label="Bat guano" href="simple_page.html"/>
	      <topic label="Bat history" href="simple_page.html"/>
	      <topic label="Bat recycling" href="simple_page.html"/>
	</topic>
</toc>