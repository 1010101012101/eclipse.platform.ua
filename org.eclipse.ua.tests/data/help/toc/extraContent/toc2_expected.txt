<?xml version="1.0" encoding="UTF-8" ?>
<toc
      label="Samples"
      href="/org.eclipse.ua.tests/data/help/toc/extraContent/toc2.xml">
   <topic
         label="Bats FOR product: Happy Bats, WAS; version: 9.0,8.0 ; platform: z/OS;AIX"
         href="/org.eclipse.ua.tests/simple_page.html">
      <topic
            label="Caring for your fruit bat FOR Version: 9.0; platform: z/OS; product: WAS"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Cleaning your fruit bat FOR Version: 8.0; platform: AIX; product: Happy Bats"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Feeding your bat"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Bat sonar"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Bat guano"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Bat history"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Bat recycling"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
   </topic>
   <topic
         label="Bats FOR product: Happy Bats"
         href="/org.eclipse.ua.tests/simple_page.html">
      <topic
            label="Feeding your bat"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Bat sonar"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Bat guano"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Bat history"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
      <topic
            label="Bat recycling"
            href="/org.eclipse.ua.tests/simple_page.html">
      </topic>
   </topic>
</toc>